lib/active_shipping/shipment_event.rb

```diff
@@ -2,14 +2,18 @@ module ActiveShipping
   class ShipmentEvent
     attr_reader :name, :time, :location, :message, :type_code
 
-    def initialize(name, time, location, message = nil, type_code = nil)
-      @name, @time, @location, @message, @type_code = name, time, location, message, type_code
+    def initialize(name, time, location, message = nil, type_code = nil, timezone_ambiguous = true)
```

Comment:
Looks like Canada Post _does_ use it though. https://github.com/Shopify/active_shipping/blob/master/lib/active_shipping/carriers/canada_post_pws.rb#L325